update-translations.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
```

Comment:
I like to use `-eu` to catch commands exiting non-zero, and undefined variables.